tests/console-backend-service/internal/domain/k8s/k8s.go

```diff
@@ -2,13 +2,20 @@ package k8s
 
 import (
 	jsonencoder "encoding/json"
+	"fmt"
+	"log"
 
+	"github.com/pkg/errors"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 type json map[string]interface{}
 
+const (
+	updateRetries = 5
```

Comment:
Maybe it should be also declared in `pkg` as constant and used where the `retry` function is used